src/lib/spinel/radio_spinel.hpp

```diff
@@ -569,6 +569,22 @@ class RadioSpinel : private Logger
      */
     otError Receive(uint8_t aChannel);
 
+    /**
+     * Schedule a radio reception window at a specific time and duration.
+     *
+     * @param[in]  aWhen      The receive window start time in the local
+     *                        radio clock, see `otPlatRadioGetNow`. The radio
+     *                        receiver SHALL be on and ready to receive the first
+     *                        symbol of a frame's SHR at the window start time.
+     * @param[in]  aDuration  The receive window duration, in microseconds, as
+     *                        measured by the local radio clock.
+     * @param[in]  aChannel   The channel to use for receiving.
+     *
+     * @retval OT_ERROR_NONE          Successfully scheduled the reception.
+     * @retval OT_ERROR_INVALID_STATE The radio was disabled.
+     */
+    otError ReceiveAt(uint64_t aWhen, uint32_t aDuration, uint8_t aChannel);
```

Comment:
I don't see this new method being called (from host) in this PR.
I think you may be missing the implmenation of `otPlatRadioReceiveAt()` on host side in `posix/platform/radio.cpp`?

Reply:
yes. This commit focuses on the rcp capability. the posix platform integration can be a separate one, which should have some posix platform tests cover the change, thoughts?

Reply:
It is up to you if you want to add it in a separate PR/commit. I noticed `RadioSpinel::ReceiveAt()` is added here but not used except in the test, but that's fine.